chapters/07-build-management.Rmd

```diff
@@ -6,7 +6,43 @@
 - Pre-push running test, build, and check
 - Continuous integration using GitHub Actions
 
-<!-- NOTE: Preferably something linked to working towards completing the project assignments, maybe to be used as a milestone? -->
+## Learning Objectives 
+
+* Acquaint with the devtools functionality needed to manage an R package infrastructure
+* Describe the CRAN checks' process and the different components
+* Understand and set up the workflow for an R package validation
+* Deploy a GitHub Action that runs a continuous integration workflow
```

Comment:
I like learning objectives to use verbs that almost describe the way they could be assessed, i.e. be measurable - "aquaint" and "understand" are a little too non-specific.  How about:
```suggestion
* Describe the functionality devtools provides to manage an R package infrastructure
* Explain the CRAN checks' process and the different components
* Implement the workflow to validate an R package locally
* Deploy a GitHub Action that runs a continuous integration workflow
```